conf/broker.conf

```diff
@@ -461,6 +461,10 @@ httpMaxRequestSize=-1
 
 ### --- BookKeeper Client --- ###
 
+# Metadata service uri that bookkeeper is used for loading corresponding metadata driver
+# and resolving its metadata service location.
+bookkeeperServiceUri=
```

Comment:
how about `bkLedgerRootPath`? This seems be more directly. 
`bookkeeperServiceUri` may be easy confused with the bookie service url(e.g. "localhost:3181").

Reply:
`bkMetadataServiceUri` seems better?

Reply:
I will add an example here to make it more clear.